projects/js-packages/publicize-components/src/components/share-buttons/useAvailableNetworks.tsx

```diff
@@ -0,0 +1,32 @@
+import { SocialIconSlug } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import { useMemo } from 'react';
+
+type Network = {
+	label: string;
+	networkName: SocialIconSlug;
+	url: string;
+};
+
+/**
+ * Returns the list of available networks.
+ *
+ * @returns {Array<Network>} The list of available networks.
+ */
+export function useAvailableNetworks() {
+	return useMemo< Array< Network > >(
+		() => [
+			{
+				label: __( 'Twitter', 'jetpack' ),
```

Comment:
Probably this is okay for now, I wonder when do we have rename all these to X or Twitter/X

Reply:
I wanted to be consistent with other parts of Jetpack for now.